src/short_id.rs

```diff
@@ -123,6 +159,8 @@ impl Default for Descriptor {
 }
 
 impl Descriptor {
+    /// verifies if Descriptor type has valid properties otherwise returns
```

Comment:
If possible pls uppercase the first letters of doc comments (except error variants)

Reply:
Thanks for clarifying that, should be ok now :+1: